docs/Deploying-Pega-on-EKS.md

```diff
@@ -0,0 +1,578 @@
+
+   ```yaml
+   helm install mypega-EKS-demo pega/pega --namespace mypega-EKS-demo --values pega.yaml --set global.actions.execute=install-deploy
+   ```
+
+   For subsequent Helm installs, use the command `helm install mypega-EKS-demo pega/pega --namespace mypega-EKS-demo` to deploy Pega Platform and avoid another Pega Platform installation.
+
+   A successful Pega deployment immediately returns details that show progress for your `mypega-EKS-demo` deployment.
+
+5. Refresh the Kubernetes dashboard that you opened in the previous section. If you closed the dashboard, start the proxy server for the Kubernetes dashboard and then relaunch the web browser.
+
+6. In the dashboard, in **Namespace** select the `mypega-EKS-demo` view and then click on the **Pods** view. Initially, you can some pods have a red status, which means they are initializing:
+
+    ! [](media/dashboard-mypega-pks-demo-install-initial.png)
+
+    Note: A deployment takes about 15 minutes for all resource configurations to initialize; however a full Pega Platform installation into the database can take up to an hour.
+
+    To follow the progress of an installation, use the dashboard. For subsequent deployments, you do not need to do this. Initially, while the resources make requests to complete the configuration, you will see red warnings while the configuration is finishing, which is expected behavior.
+
+7. To view the status of an installation, on the Kubernetes dashboard, select **Jobs**, locate the **pega-db-install** job, and click the logs icon on the right side of that row.
+
+    After you open the logs view, you can click the icon for automatic refresh to see current updates to the install log.
+
+8. To see the final deployment in the Kubernetes dashboard after about 15 minutes, refresh the `mypega-EKS-demo` namespace pods.
+
+    ! [](media/f7779bd94bdf3160ca1856cdafb32f2b.png)
+
+    A successful deployment does not show errors across the various workloads. The `mypega-EKS-demo` Namespace **Overview** view shows charts of the percentage of complete tiers and resources configurations. A successful deployment has 100% complete **Workloads**.
+
+    ! [](media/0fb2d07a5a8113a9725b704e686fbfe6.png)
+
+## Logging in to Pega Platform – 10 minutes
+
+After you complete your deployment, as a best practice, associate the host name of the pega-web tier ingress with the DNS host name that the deployment load balancer assigned to the tier during deployment. The host name of the pega-web tier ingress used in this demo, **eks.web.dev.pega.io**, is set in the pega.yaml file in the following lines:
+
+```yaml
+tier:
+  - name: "web"
+
+    service:
+      # Enter the domain name to access web nodes via a load balancer.
+      #  e.g. web.mypega.example.com
+      domain: "**eks.web.dev.pega.io**"
```

Comment:
Github not bolding the domain name -- renders the stars

Reply:
removed asterisks...good call